tools/local-network/local_network.py

```diff
@@ -100,12 +99,22 @@ def start_prometheus(self, log_branch, grafana_password, hosts):
         print(f'Starting prometheus, branch={log_branch}')
         os.mkdir(os.path.join(WORK_DIR, 'prometheus'))
         template = open(os.path.join(PROJECT_DIR, 'tools', 'local-network', 'prometheus.yml.template')).read()
+        # Add some stuff about grafana push if we have a grafana password
+        template_extra = '''
+remote_write:
+- url: https://prometheus-us-central1.grafana.net/api/prom/push
+  basic_auth:
+    username: 8687
+    password: ${GRAFANA_PASSWORD}
+        '''.replace('${GRAFANA_PASSWORD}', grafana_password) if grafana_password else ""
```

Comment:
it's a bit janky to remove this stuff from the .template file but I didn't see a simpler way to accomplish this

i'm not sure the grafana push stuff is really important to us anymore, we can just use prometheus directly when doing local networks

Reply:
I need to be able to start Prometheus even if i don't have a grafana password

I guess i could check what happens if you put a junk password in, maybe it fails authentication and then continues running normally

🤔

Reply:
oh I misinterpreted the code I was thinking this was setting the password to `""` but it's setting the `template_extra` to ""